src/slider/Slider.ts

```diff
@@ -126,7 +126,7 @@ export default class Slider extends SliderBase<SliderProperties> {
 		const outputNode = output ? output(value) : value + '';
 
 		const slider = v('div', {
-			classes: this.classes(css.inputWrapper),
+			classes: this.classes().fixed(css.inputWrapper),
```

Comment:
I think we should only add a themeable class then there's something to be themed. Ie `inputWrapper` does not need a themeable class in this case.